rpa-mapper/src/main/java/uk/gov/hmcts/cmc/rpa/mapper/SealedClaimJsonMapper.java

```diff
@@ -35,7 +42,7 @@ public JsonObject map(Claim claim) {
             .add("amountWithInterest", claim.getAmountWithInterest().orElse(null))
             .add("submitterEmail", claim.getSubmitterEmail())
             .add("claimants", mapClaimants(claim.getClaimData().getClaimants()))
-            .add("defendants", mapDefendants(claim.getClaimData().getDefendants()))
+            .add("defendants", defendantMapper.map(claim.getClaimData().getDefendants().stream().findFirst().orElseThrow(IllegalStateException::new)))
```

Comment:
On the second though `uk.gov.hmcts.cmc.rpa.mapper.DefendantJsonMapper#map(uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails)` should either accept list of party details or return `JsonObject`. Method that accepts element and returns array of elements with that single element inside is confusing.

Reply:
Reverting back with the view that when multi party is implemented this will require minimal change.

Reply:
Please change the mapper API so that we have either:
- map(element: party): json
- map(element: party[]): json[]

Having `map(element: party): json[]` does not make sense.

I hope this pseudo code is readable enough.